staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -509,11 +509,6 @@ func NewKubectlCommand(in io.Reader, out, err io.Writer) *cobra.Command {
 	}
 
 	flags := cmds.PersistentFlags()
-	flags.SetNormalizeFunc(cliflag.WarnWordSepNormalizeFunc) // Warn for "_" flags
-
-	// Normalize all flags that are coming from other packages or pre-configurations
-	// a.k.a. change all "_" to "-". e.g. glog package
-	flags.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)
```

Comment:
You can remove the `flags.SetNormalizeFunc(cliflag.WarnWordSepNormalizeFunc) // Warn for "_" flags` IMO, but you should keep this here. Otherwise no flag with '_' will be normalized, and you will have warnings even for flags you are not using

Reply:
If remove `flags.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)` and keep `flags.SetNormalizeFunc(cliflag.WarnWordSepNormalizeFunc)`. I will receive warning what I mentioned above. And these warning comes from `klog.InitFlags`, not from user. This is not friendly to user.
I think `flags.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)` and `flags.SetNormalizeFunc(cliflag.WarnWordSepNormalizeFunc)` does the same things. But the latter will cause confusion when using kubectl.

Reply:
In fact, I modified three lines:
`flags.SetNormalizeFunc(cliflag.WarnWordSepNormalizeFunc) // Warn for "_" flags`
`flags.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)`
`cmds.SetGlobalNormalizationFunc(cliflag.WarnWordSepNormalizeFunc)`
Because I think these codes are duplicated. So only keep the last one is ok, and I don't want to see these warnings every time. I delete line `flags.SetNormalizeFunc(cliflag.WarnWordSepNormalizeFunc) // Warn for "_" flags` and `flags.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)`, modify the param of `cmds.SetGlobalNormalizationFunc(cliflag.WarnWordSepNormalizeFunc)`.